app/views/pages/home.html.erb

```diff
@@ -1,11 +1,12 @@
 <section class="demo">
+  <% @lexer = Rouge::Lexer.all.sample %>
```

Comment:
I'd probably assign this in a controller. The weird bit now is we're using [high_voltage](https://github.com/thoughtbot/high_voltage) who's controller is wrapped up in a gem. It can be [overridden](https://github.com/thoughtbot/high_voltage#override), but at this point it might make sense to remove it and make a dedicated pages controller.

Whatever the case, the controller action would just have:

```ruby
@lexer = Rouge::Lexer.all.sample
```

Then the view can call the short instance methods `demo` and `tag` like you have already.